topicgenerator/results/generated-topics_CLAUDE_3_HAIKU_20240926T154233Z.csv

```diff
@@ -0,0 +1,24 @@
+trsId,version,descriptorUrl,descriptorChecksum,isTruncated,promptTokens,completionTokens,cost,finishReason,aiTopic
+"#workflow/github.com/iwc-workflows/sars-cov-2-pe-illumina-artic-variant-calling/COVID-19-PE-ARTIC-ILLUMINA",main,https://raw.githubusercontent.com/iwc-workflows/sars-cov-2-pe-illumina-artic-variant-calling/main//pe-artic-variation.ga,dcc2761eb35156d7d09479112daf089439774fc29938f02cb6ee8cda87906758,false,24595,43,0.0062025,end_turn,"Trim ARTIC primer sequences, realign reads, call and filter variants, annotate variants, and apply a strand-bias soft filter to the final annotated variants."

```

Comment:
How come some of the verbs have `s` and some don't? `Trim` here, but `Filters` on the next line? I guess it depends on the content?

I'd like it if they were all the same, but that's probably just me and probably isn't worth the effort and/or may not even make sense.

Reply:
Adding this to the prompt made it more consistent :slightly_smiling_face: